input/2D/relaxation/replicated/poly_mps_ic_onestep.i

```diff
@@ -0,0 +1,332 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 125
+  ny = 125
+  xmin = 0
+  xmax = 20000
+  ymin = 0
+  ymax = 20000
+[]
+
+[GlobalParams]
+  op_num = 12
+  var_name_base = etam
+  int_width = 480
+[]
+
+[UserObjects]
+  [./voronoi]
+    type = PolycrystalVoronoi
+    rand_seed = 1258
+    file_name = grains15_seed486.txt
```

Comment:
Out of curiosity, can this use the same thing as the parallel version?  `file_name = grains12.txt`

Reply:
Yes, the input files can use any of the .txt files for generating the grains structures. I used two different files just for the demonstration.